src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.html

```diff
@@ -22,55 +22,64 @@ <h3 class="wrapper-title">ДОДАТИ ДОСЯГНЕННЯ</h3>
         </app-validation-hint>
 
         <label class="step-label">Учасники<span class="step-required">*</span></label>
-        <mat-form-field appearance="none">
-          <mat-select multiple disableOptionCentering formControlName="childrenIDs" 
-            panelClass="dropdown-panel" class="step-input">
-            <mat-select-trigger>
-              <mat-chip-list>
-                <mat-chip 
-                  *ngFor="let child of AchievementFormGroup.controls['childrenIDs'].value"
-                  [removable]="true" (removed)="onRemoveItem(child, 'childrenIDs')">
-                  {{child.lastName}} {{child.firstName}} {{child.middleName}}
-                  <mat-icon matChipRemove>cancel</mat-icon>
-                </mat-chip>
-              </mat-chip-list>
-            </mat-select-trigger>
-            <mat-option *ngFor="let child of children$"   
-              [value]="child" class="dropdown-option">
-              {{child.lastName}} {{child.firstName}} {{child.middleName}}
-            </mat-option>
-          </mat-select>
-        </mat-form-field>
-        <app-validation-hint 
-          [validationFormControl]="AchievementFormGroup.get('childrenIDs')"
-          [isTouched]="AchievementFormGroup.get('childrenIDs').touched"
-        >
-        </app-validation-hint>
+        <ng-container *ngIf="approvedChildren.totalAmount; else noChildren"> 
```

Comment:
add another wrapper that checks for approvedChildren, it means new data is loaded, so we won't have such 
![image](https://user-images.githubusercontent.com/27493959/180227467-ab649cfd-9c2b-426e-a516-6b30af1b2cf3.png)

Reply:
![image](https://user-images.githubusercontent.com/27493959/180230031-830e2390-fc5f-4e85-b2ed-94c7ac52215d.png)